Cargo.toml

```diff
@@ -50,6 +54,10 @@ ron = "0.8"
 serde_cbor = "0.11"
 hex = { version = "0.4", features = ["serde"] }
 
+# usbip
+trussed-usbip = { git = "https://github.com/trussed-dev/pc-usbip-runner", default-features = false, features = ["ccid"]}
+usbd-ccid = { git = "https://github.com/Nitrokey/nitrokey-3-firmware", features = ["highspeed-usb"]}
```

Comment:
I think it would be better to add a `highspeed-usb` feature to `trussed-usbip`.  Otherwise making sure that this points to the same `usbd-ccid` as `trussed-usbip` will be painful until we have proper releases.